tests/graph_test.py

```diff
@@ -1,45 +1,25 @@
-from mythril.analysis.callgraph import generate_graph
 from mythril.mythril import MythrilAnalyzer, MythrilDisassembler
 from mythril.ethereum import util
 from mythril.solidity.soliditycontract import EVMContract
 from tests import (
-    BaseTestCase,
     TESTDATA_INPUTS,
-    TESTDATA_OUTPUTS_EXPECTED,
-    TESTDATA_OUTPUTS_CURRENT,
 )
-import re
 
 
-class GraphTest(BaseTestCase):
-    def test_generate_graph(self):
-        for input_file in TESTDATA_INPUTS.iterdir():
-            output_expected = TESTDATA_OUTPUTS_EXPECTED / (
-                input_file.name + ".graph.html"
-            )
-            output_current = TESTDATA_OUTPUTS_CURRENT / (
-                input_file.name + ".graph.html"
-            )
+def test_generate_graph(Z):
+    for input_file in TESTDATA_INPUTS.iterdir():
+        if input_file.name != "origin.sol.o":
+            continue
+        contract = EVMContract(input_file.read_text())
+        disassembler = MythrilDisassembler()
+        disassembler.contracts.append(contract)
+        analyzer = MythrilAnalyzer(
+            disassembler=disassembler,
+            strategy="dfs",
+            execution_timeout=5,
+            max_depth=30,
+            address=(util.get_indexed_address(0)),
+        )
 
-            contract = EVMContract(input_file.read_text())
-            disassembler = MythrilDisassembler()
-            disassembler.contracts.append(contract)
-            analyzer = MythrilAnalyzer(
-                disassembler=disassembler,
-                strategy="dfs",
-                execution_timeout=5,
-                max_depth=30,
-                address=(util.get_indexed_address(0)),
-            )
+        analyzer.graph_html(transaction_count=1)
 
-            html = analyzer.graph_html(transaction_count=1)
-            output_current.write_text(html)
-
-            lines_expected = re.findall(
-                r"'label': '.*'", str(output_current.read_text())
-            )
-            lines_found = re.findall(r"'label': '.*'", str(output_current.read_text()))
```

Comment:
I always wondered how these tests work without replacing the files, apparently it's comparing [] and [] as there is no label in most of the testable circumstances.